src/content/docs/extensions/vector.mdx

```diff
@@ -0,0 +1,270 @@
+    2
+)
+RETURN nn.title ORDER BY distance;
+```
+The result is as follows:
+```
+┌───────────────────┐
+│ nn.title          │
+│ STRING            │
+├───────────────────┤
+│ The Quantum World │
+│ Learning Machines │
+└───────────────────┘
+```
+
+The results make sense! We found books whose titles are semantically similar to
+both "quantum physics" and "machine learning".
+
+### Index Management
+
+#### Drop an Index
+To remove a vector index, use the `DROP_VECTOR_INDEX` function:
+
+```cypher
+// Remove an existing vector index
+CALL DROP_VECTOR_INDEX('Book', 'title_vec_index');
+```
+
+#### List All Indexes
```

Comment:
I would document this in the CALL page together with other SHOW_XXX functions.

Reply:
We already have SHOW_INDEXES documented there.